gateway/src/apicast/policy/mtls/mtls.lua

```diff
@@ -0,0 +1,41 @@
+local policy = require('apicast.policy')
+local _M = policy.new('mtls')
+local X509 = require('resty.openssl.x509')
+local errors = require('apicast.errors')
+local re_split = require('ngx.re').split
+
+local mt = {
+  __index = _M
+}
+
+local function encode(bytes)

```

Comment:
This should explain what it encodes to. It should be `base64url-encoded` format.

And then why not to use https://github.com/openresty/lua-resty-core/blob/master/lib/ngx/base64.md#encode_base64url ?

Reply:
Changed.